server/src/jobs/jobController.js

```diff
@@ -0,0 +1,37 @@
+import Job from './jobModel';
+import request from 'request';
+
+//Add a job
```

Comment:
instead of something bland like add a job, better to prefix controller methods with the api endpoint it hooks up to. Eg:

POST method for this api route /slack/jobs/create (or whatever it is)